timed/projects/views.py

```diff
@@ -113,7 +129,9 @@ def get_queryset(self):
             if models.CustomerAssignee.objects.filter(
                 user=user, is_customer=True
             ).exists():
-                return queryset.filter(Q(customer__assignees=user))
+                return queryset.filter(
+                    Q(customer__assignees=user, customer_visible=True)
```

Comment:
Do you really need a Q object here?

Reply:
I need the assignees for the filter, that's why I used a Q object here.
Is there another solution/idea you had in mind?

Reply:
Wouldn't this work also without the Q object? But it could also be that I am mistaken. :-)

Reply:
It does work without the Q object. 
Updated the PR.

Reply:
Nice! :+1: